src/gcd.c

```diff
@@ -3,8 +3,15 @@
 const int n = 1000;
 
 unsigned int Euclidean_gcd_rec(unsigned int x, unsigned int y){
+  while(x != 0){
+    unsigned int z = x;
+     x = y % x;
+     y = z;
+  }
+  return y;
```

Comment:
```suggestion
  while(x != 0){
    unsigned int z = x;
     x = y % x;
     y = z;
  }
  return y;
```
Conversation に普通の投稿としてコメントしてしまいましたが，変更して push しなおす必要がある点なので request changes として投稿しなおします．

Reply:
pushのやり方がわからないので教えてもらってもよろしいですか

Reply:
一回目に `push` したのと同様にしてください。覚えていなければ説明の動画を確認してください。 `git add` からやればいいです。